app/javascript/miq-redux/store.js

```diff
@@ -9,6 +9,21 @@ const initialState = {};
 
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 
+const notificationInitialState = {
+  unreadCount: 0,
+};
+
+const nPrefix = '@@notifications/';
+
+const notificationReducer = (state = notificationInitialState, action) => {
```

Comment:
wrong file for this, our store implementation should probably not include notification-specific reducers.

Don't we already have a way for a react component to register a reducer and initial state, instead of hardcoding it in our store?

Reply:
I think that stores like this which have global scope and will be always present no matter what screen is on display, can be directly in initial state. Yes maybe we can have some file with global stores but i see no problem having them in initial state. You will add them only once and you never want to remove them.

Reply:
Oh, my issue is more with this happening in a surprising place, not with having the data be part of the initial state, that part is fine.

(It's about where the code lives, not when the code gets called.)

But, if we currently don't have a mechanism to do that, no issue here, no point creating one in this PR :).